sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/util/BuilderHelper.java

```diff
@@ -3,12 +3,16 @@
 
 package com.azure.storage.file.share.implementation.util;
 
+import com.azure.core.credential.AzureSasCredential;
```

Comment:
I may not understand the structure of the Java codebase very well, but is this a hand-written class?  If so, why is it an implementation folder?

Reply:
Classes all under the implementation/util folder are all hand-written.